submissions/jeffersonMormah/css/contact.css

```diff
@@ -218,6 +221,7 @@ footer p{
         float: none;
 }
 main, aside {
+    width: 70%;
```

Comment:
You are declaring this twice??? @Jeffbuchi

Reply:
Rectified